### PR TITLE
fixing for style changes

### PR DESCRIPTION
**Pull request by sdwheeler**
Fixing exe references per our style discussion

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Learning-PowerShell-Names.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Learning-PowerShell-Names.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C29%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201808291929465907-2869%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=y9JYRPIj6LZVOJigZPFWUEaHP83kEMr72NoKkrRV4gA%3D&st=2018-08-29T19%3A45%3A51Z&se=2018-09-29T19%3A50%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md [hunk 1]
```diff
@@ -82,19 +83,19 @@ variables using the variable drive:
 Get-ChildItem variable:
 ```
 
-## Using Cmd.exe variables
+## Using cmd.exe  variables
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md]: Remove bonus space after `cmd.exe`

File: reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md [hunk 2]
```diff
@@ -82,19 +83,19 @@ variables using the variable drive:
...
 
 PowerShell can use the same environment variables available to any Windows process, including
-Cmd.exe. These variables are exposed through a drive named `env:`. You can view these variables by
-typing the following command:
+**cmd.exe** . These variables are exposed through a drive named `env:`. You can view these
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md]: Remove space after `cmd.exe` before period.

File: reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md [hunk 3]
```diff
@@ -16,15 +16,15 @@ function named `Clear-Host` that clears the output window. You can type either t
 `clear` alias at a command prompt. PowerShell interprets these aliases and runs the
 `Clear-Host` function.
 
-This feature helps users to learn PowerShell. First, most Cmd.exe and Unix users have a large
+This feature helps users to learn PowerShell. First, most **cmd.exe** ** and Unix users have a large
 repertoire of commands that users already know by name. The PowerShell equivalents may not produce
 identical results. However, the results are close enough that users can do work without knowing the
 PowerShell command name. "Finger memory" is another major source of frustration when learning a
-new command shell. If you have used Cmd.exe for years, you might reflexively type the `cls` command
+new command shell. If you have used **cmd.exe**  for years, you might reflexively type the `cls` command
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md]: Remove bonus space after `cmd.exe`

File: reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md [hunk 4]
```diff
@@ -16,15 +16,15 @@ function named `Clear-Host` that clears the output window. You can type either t
...
 to clear the screen. Without the alias for `Clear-Host`, you receive an error message and won't
 know what to do to clear the output.
 
-The following list shows a few of the common Cmd.exe and Unix commands that you can use in
+The following list shows a few of the common **cmd.exe**  and Unix commands that you can use in
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md]: Remove bonus space after `cmd.exe`

File: reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md [hunk 5]
```diff
@@ -16,15 +16,15 @@ function named `Clear-Host` that clears the output window. You can type either t
 `clear` alias at a command prompt. PowerShell interprets these aliases and runs the
 `Clear-Host` function.
 
-This feature helps users to learn PowerShell. First, most Cmd.exe and Unix users have a large
+This feature helps users to learn PowerShell. First, most **cmd.exe** ** and Unix users have a large
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md]: Remove extra asterisks after `cmd.exe`

File: reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md [hunk 6]
```diff
@@ -25,15 +25,15 @@ syntax.
...
+Interfaces such as **cmd.exe** don't provide a way for you to directly extend the built-in command
+set. You can create external command-line tools that run in **cmd.exe** . But these external tools
+don't have services, such as Help integration. **cmd.exe** doesn't automatically know that these
+external tools are valid commands.
 
 The native commands in PowerShell are known as *cmdlets* (pronounced command-lets). You can create
 your own cmdlets modules and functions using compiled code or scripts. Modules can add cmdlets and
 providers to the shell. PowerShell also supports scripts that are analogous to UNIX shell scripts
-and Cmd.exe batch files.
+and **cmd.exe**  batch files.
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md]: Remove bonus space after `cmd.exe`

File: reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md [hunk 7]
```diff
@@ -25,15 +25,15 @@ syntax.
 
 ## The command family is extensible
 
-Interfaces such as Cmd.exe don't provide a way for you to directly extend the built-in command set.
-You can create external command-line tools that run in Cmd.exe. But these external tools don't have
-services, such as Help integration. Cmd.exe doesn't automatically know that these external tools are
-valid commands.
+Interfaces such as **cmd.exe** don't provide a way for you to directly extend the built-in command
+set. You can create external command-line tools that run in **cmd.exe** . But these external tools
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md]: Remove space after `cmd.exe` before period.

File: reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md [hunk 8]
```diff
@@ -74,7 +74,7 @@ It is more useful to look at a reduced set of commands.
 > [!NOTE]
 > The asterisk (\*) is used for wildcard matching in PowerShell command arguments. The \* means
 > "match one or more of any characters". You can type `Get-Command a*` to find all commands that
-> begin with the letter "a". Unlike wildcard matching in Cmd.exe, PowerShell's wildcard will also
+> begin with the letter "a". Unlike wildcard matching in **cmd.exe** , PowerShell's wildcard will also
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md]: Remove space after `cmd.exe` before comma.

File: reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md [hunk 9]
```diff
@@ -24,10 +24,10 @@ Cmdlet          Add-Type                3.1.0.0    Microsoft.PowerShell.Utility
 ...
 ```
 
-This output looks a lot like the Help output of Cmd.exe: a tabular summary of internal commands. In
+This output looks a lot like the Help output of **cmd.exe** : a tabular summary of internal commands. In
 the excerpt of the `Get-Command` command output shown above, every command shown has a CommandType
 of Cmdlet. A cmdlet is PowerShell's intrinsic command type. This type corresponds roughly to
-commands like `dir` and `cd` in Cmd.exe or the built-in commands of Unix shells like bash.
+commands like `dir` and `cd` in **cmd.exe**  or the built-in commands of Unix shells like bash.
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md]: Remove bonus space after `cmd.exe`

File: reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md [hunk 10]
```diff
@@ -24,10 +24,10 @@ Cmdlet          Add-Type                3.1.0.0    Microsoft.PowerShell.Utility
 ...
 ```
 
-This output looks a lot like the Help output of Cmd.exe: a tabular summary of internal commands. In
+This output looks a lot like the Help output of **cmd.exe** : a tabular summary of internal commands. In
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md]: Remove space after `cmd.exe` before colon.

File: reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md [hunk 11]
```diff
@@ -82,19 +83,19 @@ variables using the variable drive:
...
+variables by typing the following command:
 
 ```powershell
 Get-ChildItem env:
 ```
 
 The standard `*-Variable` cmdlets aren't designed to work with environment variables. Environment
 variables are accessed using the `env:` drive prefix. For example, the **%SystemRoot%** variable in
-Cmd.exe contains the operating system's root directory name. In PowerShell, you use
+**cmd.exe**  contains the operating system's root directory name. In PowerShell, you use
```
Comment (DCtheGeek) [inline reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md]: Remove bonus space after `cmd.exe`

File: reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md (hunk 5 above)
Comment (sdwheeler) [reply reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md]: Fixed find/replace error

File: reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md (hunk 7 above)
Comment (sdwheeler) [reply reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md]: Fixed find/replace error

File: reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md (hunk 6 above)
Comment (sdwheeler) [reply reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md]: Fixed find/replace error

File: reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md (hunk 8 above)
Comment (sdwheeler) [reply reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md]: Fixed find/replace error

File: reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md (hunk 10 above)
Comment (sdwheeler) [reply reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md]: Fixed find/replace error

File: reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md (hunk 11 above)
Comment (sdwheeler) [reply reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md]: Fixed find/replace error

File: reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md (hunk 9 above)
Comment (sdwheeler) [reply reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md]: Fixed find/replace error

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Learning-PowerShell-Names.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Learning-PowerShell-Names.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C29%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201808292027111685-2869%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zPsG2%2FrxOrEEOR8BD%2B6%2FVoTR8%2FXx6emoaPi87JrcRl8%3D&st=2018-08-29T20%3A38%3A16Z&se=2018-09-29T20%3A43%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
Prematurely removed the branch.

**Comment by sdwheeler** [comment]
#sign-off

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/getting-information-about-commands?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Learning-PowerShell-Names.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Learning-PowerShell-Names.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/learning-powershell-names?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Understanding-Important-PowerShell-Concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/understanding-important-powershell-concepts?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Using-Familiar-Command-Names.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-familiar-command-names?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |
[reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-stylefix/reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-3.0&branch=pr-en-us-2869) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-4.0&branch=pr-en-us-2869) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-5.0&branch=pr-en-us-2869) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-5.1&branch=pr-en-us-2869) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/getting-started/fundamental/using-variables-to-store-objects?view=powershell-6&branch=pr-en-us-2869) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C29%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201808292109445168-2869%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=o2pSb7m%2FenxkfmQYcCRV8IwLwAuJS312xdcL8lhXMrI%3D&st=2018-08-29T21%3A21%3A02Z&se=2018-09-29T21%3A26%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**